wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -718,7 +722,74 @@ public boolean uploadFileLocalDropZone(File file) throws Exception
 
         return uploadFile(file, fetchLocalDropZones[0]);
     }
+    public boolean upLoadLandingZone(File uploadFile,DropZone dropZone) {
```

Comment:
It's not immediately obvious what this method does when compared to uploadFile(File file, DropZone dropZone).
Please rename the method so it's self documenting, also a block comment describing the method would help.